terraform/budget.tf

```diff
@@ -0,0 +1,25 @@
+locals {
+    total_monthly_budget_amount = "600"
+    total_monthly_budget_threshold_percentage = "100"
+    total_monthly_budget_subscribers = ["Pay.Digital@gov.bc.ca"]
+}
+
+
+
+resource "aws_budgets_budget" "budget-monthly" {
+    name = "Total Monthly Budget (${local.environment})"
+    budget_type = "COST"
+    limit_amount = local.total_monthly_budget_amount
+    limit_unit = "USD"
```

Comment:
I found conflicting answers on whether we can ever change this lol - AWS docs 😮‍💨